db/migrate/20150313054707_create_translations.rb

```diff
@@ -0,0 +1,9 @@
+class CreateTranslations < ActiveRecord::Migration
+  def change
+    create_table :translations do |t|
+      t.datetime :time_assigned
+
```

Comment:
Yeah, without ```belongs_to :user``` and ```belongs_to :video``` a translation entry won't be to access which Video and User owns it. So ```@translation.user``` won't work, but ```@video.translations``` would still return an array of translations I believe.